rosserial_client/src/ros_lib/ros/node_handle.h

```diff
@@ -64,6 +64,10 @@ class NodeHandleBase_
 namespace ros
 {
 
+const int SPIN_OK = 0;
+const int SPIN_ERR = -1;
+const int SPIN_TIMEOUT = -2;
```

Comment:
There's a case to be made that these should be static const int members of NodeHandler, but it's probably fine for them just to be here.